SCF-Assignment-3/Question3/AreaCalculate.java

```diff
@@ -0,0 +1,88 @@
+package Question3;
+
+import java.util.Scanner;
+
+/**
+ * this class contains main class and will call the functions of area and will
+ * print the respective values.
+ * 
+ * @author Arjita
+ *
+ */
+public class AreaCalculate {
+	/**
+	 * 
+	 * @param objectArea
+	 *            object of area class
+	 * @param areaOfTriangle
+	 *            will store the return value of triangle area
+	 * @param areaOfRectangle
+	 *            will store the return value of rectangle area
+	 * @param areaOfSquare
+	 *            will store the return value of square area
+	 * @param areaOfCircle
+	 *            will store the return value of circle area
+	 */

```

Comment:
I cannot see any parameters like defined in main class.
We don't define the local variable documentation here.